server/src/main/java/com/linecorp/centraldogma/server/internal/storage/repository/git/GitRepository.java

```diff
@@ -1224,15 +1274,21 @@ static void doRefUpdate(org.eclipse.jgit.lib.Repository jGitRepository, RevWalk
         final CompletableFuture<Revision> future = new CompletableFuture<>();
 
         normalize(lastKnownRevision).thenAccept(normLastKnownRevision -> {
```

Comment:
Not related to your fix. How about using `normalizeNow()` here?

Reply:
It's actually more convenient to use `normalize()` here because otherwise we will need additional `try-catch` block here.